### ISSUE TITLE
Adapt the Traceability architecture document to the needs of WP3 and WP4

### ISSUE THREAD

**Issue by cecilebraun** [openETCS/toolchain#471]
Tasks to be done by Faiez to complete this issue:

- [ ] At least rudimentary complete all open sections in this document (marked as TODO)
- [ ] Talk to WP3 and WP4 representatives, find out what they are doing TODAY and what they want to do in the near Future and incorporate it in this document.
- [ ] Organize a review of this document, involving WP3, WP4 and WP7

------
@openETCS/toolchain 
Here is proposition for the traceability architecture:
https://github.com/openETCS/toolchain/blob/master/T7.3/TraceabilityArchitecture/oetc_WP7_traceability.pdf

Please read it and comment it in this issue. 
We will discuss it 08.01.2015

**Comment by raphaelfaudou** [comment]
Dear all,

Just joining the project on WP7 traceability, in support of ENSEEIHT partner.

I have read the last revision of traceability architecture document and I get some confusion with current figure about traceability process.

1. For me all traceability link types (satisfies, realizes, refines and defines) concern relations between engineering artefacts. I do not understand how we can put "satisfy" relations between artefact and activities. Can anyone explain?

2. Other confusion concerns what does "System model" means: is it architecture model? specification model? 
In case it is specification model, I'm not comfortable with "realize" relation between functional model and system model. Generally we get high level system functions that are "services" provided by system and that refine system requirements.
Then those functions are broken down into functions of lower complexity and we get functional architecture. 
Finally when physical architecture is identified, there are new functions (technical) that appear in order to support functional flows carried by physical interfaces. Those functions are derived from physical architecture.
For me "realization" is rather a term used for physical components (physical architecture) that realize functions (and associated functional requirements) in addition to other non functional requirements.

3. Where is physical architecture?

**Comment by raphaelfaudou** [comment]
Perhaps

**Comment by raphaelfaudou** [comment]
Dear all,

here is what I could capture from interview with WP3 leaders last friday.


Please comment if you find mistakes or disagree.

**Comment by raphaelfaudou** [comment]
Dear all,
I have completed traceability architecture document in two directions:
1. added background for reviewers in the beginning of document to introduce requirement engineering levels, models scopes, priorities, expectations for traceability (that I could capture) and first tool chain requirements.
2. completed suggested traceability architecture solutions with two other solutions more integrated.

I took the time to do a lot of diagrams in order to help understanding and provide concise illustration.

I would appreciate feedback on described context (I might be wrong in some interpretation and I'm quite new to the project) and on suggested second and third solutions.

Thanks
